hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -72,7 +72,7 @@
     // TODO Introducing this to allow disabling the spoofing checks on-demand
     // if there is a use-case that gets affected by the change. If there are no reports of misbehaviour we can remove than in
     // next release.
-    private static final boolean SPOOFING_CHECKS = parseBoolean(getProperty("hazelcast.nio.tcp.spoofing.checks", "true"));
+    private static final boolean SPOOFING_CHECKS = parseBoolean(getProperty("hazelcast.nio.tcp.spoofing.checks", "false"));
```

Comment:
Just wondering - why it was on by default? Was it some recommendation from the security audit?

Reply:
Yep. This prevents an attack that they found, exploiting direct connection to a member, without joining the cluster, which allows them to execute anything.

Reply:
I'm guessing we documented this is off by default then in some hardening guide and it can be turned on?

Reply:
Working on that, yes, will have it listed.